npm/ux/data.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export type ReData = {
+  status: string;
+  jobParams: any;
+  physicalCounts: any;
+  physicalCountsFormatted: any;
+  logicalQubit: any;
+  tfactory: any;
+  errorBudget: any;
+  logicalCounts: any;
+  new: boolean;
+  frontierEntries: FrontierEntry[];
+};
+
+export type FrontierEntry = {
+  logicalQubit: any;
+  tfactory: any;
+  errorBudget: any;
+  physicalCounts: any;
+  physicalCountsFormatted: any;
+};
+
+export function CreateReData(
```

Comment:
Should this just be called `GetFrontierEntry` and require the second param? There's little point in calling it unless you pass the second param to get the specific entry, as it just reflects the first param directly.

Reply:
Actually, ReData was used for two different purposes: 
- original binding strucutre for controls
- structure received from the RE, which was extended with FrontierEntries

Now, I have intorduced SingleEstimateResult for the former purpose. Hopefully, it would clarify data conversions.